src/plugin.js

```diff
@@ -100,7 +100,7 @@ export default (renderFn: Render) =>
           const serialized = JSON.stringify(initialState);
           const script = html`
             <script type="application/json" id="__APOLLO_STATE__">
-              ${serialized}
+              ${String(serialized)}
```

Comment:
Hmm, or is this going to cause issues with the `html` tagged template functionality? 🤔

Reply:
No, I guess `serialized` should be a string after all given the JSON.stringify call above.